src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,55 @@
+import classNames from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+import { SearchLink } from './SearchLink';
+import { getSearchWith } from '../utils/searchHelper';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const sex = searchParams.get('sex');
+  const centuries = searchParams.getAll('centuries') || [];
+
+  const onSearchFilterChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchParams(getSearchWith(
+      searchParams,
+      { query: e.target.value || null },
+    ));
+  };
+
+  const getCenturies = (newCentury: string) => {
+    if (centuries.includes(newCentury)) {
+      return centuries.filter(century => century !== newCentury);
+    }
+
+    return [...centuries, newCentury];
+  };
+
   return (
     <nav className="panel">
       <p className="panel-heading">Filters</p>
 
       <p className="panel-tabs" data-cy="SexFilter">
-        <a className="is-active" href="#/people">All</a>
-        <a className="" href="#/people?sex=m">Male</a>
-        <a className="" href="#/people?sex=f">Female</a>
+        <SearchLink
+          params={{ sex: 'all' }}
+          className={sex === 'all' || sex === null
+            ? 'is-active'
+            : ''}
```

Comment:
Pay attention to the JS library "classnames". It should be a more convenient way to manage conditional class names.